### ISSUE TITLE
Build info in docker image is not set

### ISSUE THREAD

**Issue by carlpett** [oliver006/redis_exporter#38]
The latest docker image does not seem to be built correctly, the build_info series is not populated:

```bash
$ docker run -d -P oliver006/redis_exporter:v0.9.1
Unable to find image 'oliver006/redis_exporter:v0.9.1' locally
v0.9.1: Pulling from oliver006/redis_exporter
[...]
Digest: sha256:8c9585f48075dcf6888f4bca63813052c147d13a0fb9d7b0677c38b08ac13423
Status: Downloaded newer image for oliver006/redis_exporter:v0.9.1
$ curl localhost:32771/metrics | grep version
redis_exporter_build_info{build_date="<<< filled in by build >>>",commit_sha="<<< filled in by build >>>",golang_version="go1.6.3",version="<<< filled in by build >>>"} 1
```

**Comment by carlpett** [comment]
Actually, it never seems to have worked - v0.9.0, v0.8.0 all have the wrong versions

**Comment by carlpett** [comment]
Intrestingly, the binaries available on the Releases page here on Github have the correct info. From my reading of the Circle config, they should be the same?

**Comment by oliver006** [comment]
weird, yeah, they should be. Let me review the Dockerfile, I have a suspicion it builds the binary from scratch and doesn't use the env variables to set the version correctly.

**Comment by oliver006** [comment]
This has been fixed by #43
